infra/license/license-checker.ts

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {readFileSync, writeFileSync} from 'fs';
+import {EOL} from 'os';
+
+import * as licenseJudgment from './license-judgment.json';
+import * as packageJudgment from './package-judgment.json';
+
+
+/**
+ * Argument List
+ *  - Input path for list of used licenses
+ *  - Tag name for the verification
```

Comment:
`tag` is very abstract term which could mean anything. Maybe some more explanation can help reader, e.g, purpose of this tag.

Reply:
Then how about `Execution Environment at Github Action` ? :D

Reply:
How about the following? Would it be too specific?
```
name of OS. Currently `Windows`, `Linux` or `MaxOS` are used.
```

Reply:
If we don't need to run in both `ubuntu-18.04` and `ubuntu-20.04`, It would be fine :)
Just FYI, `ubuntu-latest`, `windows-latest`, `macos-latest` are given for the argument :D